aws/amazon-ivs-react-native-player
# resize mode finishing up

- [x] Automated tests need to be written
- [x]  Doc updates
- [x] Add the resizeMode functionality to the example apps so that customers can try it and our manual QA team can trigger the different resizeModes

QA Criteria: 
 - Playground Example screen presents the video with according configuration options
 - Settings can be found once press the top right corner icon
 - Settings modal presents the list of available configurations along with the `resizeMode` option
 - SDK offers 3 kinds of resize modes: `Aspect Zoom`, `Aspect Fit` and `Aspect Fill` (https://github.com/arekkubaczkowski/amazon-ivs-react-native-player/blob/feat/resize-mode-finishing-up/docs/ivs-player-reference.md#resizemode-optional)
 - Selected option should be applied immediately and visible on video player

https://user-images.githubusercontent.com/18654447/155139507-928e5383-864c-4cb3-a8f4-713e7332a18e.mov
